main/src/main/scala/sbt/Defaults.scala

```diff
@@ -1387,6 +1390,91 @@ object Defaults extends BuildCommon {
     }
     analysisResult.analysis
   }
+  lazy val previousAnalysis = Def.inputKey[String]("gets previous analysis")
+  lazy val previousAnalysisParser = Def.setting {
+    Parsers.StringBasic
+  }
+  lazy val previousAnalysisCommand = Command.single("previousAnalysis") { (state, definition) =>
```

Comment:
Why are we adding both an input key and a command here?

Reply:
fixed